### PR TITLE
Adds errorneous facet to processes page

### PR DESCRIPTION
**Pull request by fhaScireum**
- Fixes: SIRI-274

### CODE REVIEW

**Review by sabieber — APPROVED** [review]

**Comment by andyHa** [comment]
Why do we add an errorneos facet? :-P

**Review by andyHa — APPROVED** [review]

**Review by mkeckmkeck — APPROVED** [review]

File: src/main/resources/biz_en.properties [hunk 1]
```diff
@@ -508,6 +508,7 @@ PersonData.title = Title
 Process = Process
 Process.completed = Completed
 Process.counters = Performance
+Process.errorneous = Errorneous
```
Comment (idlira) [inline src/main/resources/biz_en.properties:511]:
```suggestion
Process.errorneous = Erroneous
```

File: src/main/java/sirius/biz/process/ProcessController.java [hunk 2]
```diff
@@ -96,6 +96,7 @@ public void processes(WebContext ctx) {
         ElasticPageHelper<Process> pageHelper = ElasticPageHelper.withQuery(query);
         pageHelper.withContext(ctx);
         pageHelper.addTermAggregation(Process.STATE, ProcessState.class);
+        pageHelper.addBooleanAggregation(Process.ERRORNEOUS);
```
Comment (idlira) [inline src/main/java/sirius/biz/process/ProcessController.java:99]: Actually ERRONEOUS ist the right word.

File: src/main/resources/biz_en.properties (hunk 1 above)
Comment (fhaScireum) [reply src/main/resources/biz_en.properties:511]: Huh, seems like the field is named wrongly

File: src/main/java/sirius/biz/process/ProcessController.java (hunk 2 above)
Comment (fhaScireum) [reply src/main/java/sirius/biz/process/ProcessController.java:99]: :-)

**Comment by fhaScireum** [comment]
> Why do we add an errorneos facet? :-P

So it is easier to find erroneous processes :-P (requested by OXOMI support)

**Pull request merged**